t/05-scheduler-dependencies.t

```diff
@@ -872,4 +874,22 @@ subtest 'slepos test workers' => sub {
     ok(_job_cloned_and_related($jobBS,  $jobT),   "jobBS and jobT");
 };
 
+# This enforces order in the processing of the nodes, to test PR#1623
+my $ordered_sort = sub {
+    my ($self, $result_set, $condition, $attrs) = @_;
+    $attrs->{order_by} = {-desc => 'id'};
+    return $self->$result_set->search($condition, $attrs);
+};
+
+my $unordered_sort = *OpenQA::Schema::Result::Jobs::search_for;
+my $tb             = Test::More->builder;
```

Comment:
Not sure why needed, i think you could also just use subtest
```perl subtest => coderef
```

Reply:
Done. Removed the testbuilder